src/pages/docs-template.js

```diff
@@ -12,7 +12,12 @@ class DocsTemplate extends React.Component {
     const { content, data } = doc;
     const { title, scope } = data;
     return (
-      <Page withSidebar sidebarContent={sidebarContent} location={location}>
+      <Page
+        withSidebar
+        sidebarContent={sidebarContent}
+        sidebarContent={sidebarContent}
```

Comment:
Accidental doubled prop?

Reply:
Yep! I actually saw this but forgot to push up the change